lib/threema/receive.rb

```diff
@@ -57,7 +59,9 @@ def type_instance(type:, params:)
       end
 
       def classify(type)
-        Object.const_get(class_name(type))
+        return Object.const_get(class_name(type)) if Object.const_defined?(class_name(type))
```

Comment:
@thorsteneckel `Object.const_get` will throw a `NameError` if that constant doesn't exist. It happens when we maintainer forgot to implement a constant for particular message type. That was the case with e.g. `DeliveryReceipt`. Our sever would respond 500 if Threema tried to call our webhook after I open Threema and read a message.

In general, I think there should be a `default` case to avoid unsafe `const_get`. E.g. for the future if Threema adds new message types.

Reply:
We could change the name of the `NotImplementedFallback` to make it more obvious?

Reply:
I'd prefer t raise a dedicated exception (class) instead of silently "handling" not supported messages tbh. I agree `NameError` is not optimal though. The implementation should then catch the dedicated exception and handle it accordingly instead of checking the message class each time.

Reply:
I would assume that most developers are not interested to handle this error. I think it's safe to assume that the default should be ignoring unknown message types instead of crashing or manual error handling. 

If you disagree, I can refactor this PR and throw a dedicated error here. At least, it should be documented in the README how to catch that error.

Reply:
Any update on this?